.github/workflows/macos.yml

```diff
@@ -60,7 +62,15 @@ jobs:
       - name: check
         run: |
           cd _build
-          make check TESTSUITEFLAGS="--jobs=$((${NPROC}+1))"
+          make --jobs=$((${NPROC}+1)) check TESTSUITEFLAGS="--jobs=$((${NPROC}+1))" || \
+            ( make -C tests testsuite-logs-tar; exit 1 )
+
+      - name: Upload testsuite logs
+        uses: actions/upload-artifact@v3
+        if: failure()
+        with:
```

Comment:
That's too much, in 99% of the cases it is enough to upload tests/testsuite.log in case of `make check` errors and `config.log` (included in the former) in case of configure errors).

Reply:
Ok I have changed the CI workflows to upload the logs only upon failure.

Reply:
Ok; I guess you know best whether none of the other sub-directories induce some checks.